monai/handlers/utils.py

```diff
@@ -55,71 +48,6 @@ def stopping_fn(engine: Engine):
     return stopping_fn
 
 
-def evenly_divisible_all_gather(data: torch.Tensor) -> torch.Tensor:
```

Comment:
I think this is a breaking change, but the PR's description says non-breaking?

Reply:
You are right, the default value of `concat` is a breaking change, sorry I forgot to change the PR check box.
Updated the PR description.

Thanks.